research/src/pages/toolbar.proposal.mdx

```diff
@@ -0,0 +1,72 @@
+---
+menu: Proposals
+name: Toolbar (Editor's Draft)
+path: /components/toolbar
+pathToResearch: /components/toolbar.research
+---
+
+import ToolbaAnatomy from '../components/toolbar-anatomy'
+
+## Overview
+
+A `toolbar` is a container for grouping a set of controls, such as buttons, menubuttons, or checkboxes.
+
+## Use Cases
+
+A `Toolbar` groups visually a set of controls semantically algined and it can also be ane effective way of reducing the number of tab stops
+
+## Prior Art/Examples <a href="#prior-art" id="prior-art"></a>
+
+- [Stardust](https://fluentsite.z22.web.core.windows.net/0.59.0/components/toolbar/definition)
+- [Reakit](https://reakit.io/docs/toolbar/)
+- [Radix UI](https://www.radix-ui.com/docs/primitives/components/toolbar)
```

Comment:
We have other prior art examples given, should we link out to those as well? Would we consider having more generic examples as well such as the toolbar from RoosterJS's rich text editor?